user_data/emqx_init.sh

```diff
@@ -58,15 +65,17 @@ maybe_install_from_src() {
   pushd ./
   if [ -d emqx ]; then
     echo "Find emqx source code, install from source code..."
+    install_docker
     cd emqx
     docker run --rm -i \
            -v "$PWD":/emqx \
            -w /emqx \
            -e EMQX_NAME="emqx" \
            -e HOME="/root" \
            "$EMQX_BUILDER_IMAGE" \
-           bash -c "make $EMQX_BUILD_PROFILE"
+           bash -c "make && make $EMQX_BUILD_PROFILE"
```

Comment:
I think we can remove this now that emqx/emqx#7432 is merged.

```suggestion
           bash -c "make $EMQX_BUILD_PROFILE"
```